dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -144,123 +147,125 @@ def readRecord(self,reader,writer,fileType,interfaces,rowNumber,jobId,fields):
-                rowErrorFound = True
-            return {}, reduceRow, True, False, rowErrorFound
-        return record, reduceRow, False, False, rowErrorFound
+                writer.write(["Formatting Error", ValidationError.readErrorMsg, str(row_number), ""])
+                error_list.recordRowError(job_id,self.filename,"Formatting Error",ValidationError.readError,row_number,severity_id=interfaces.validationDb.getRuleSeverityId("fatal"))
+                row_error_found = True
+            return {}, reduce_row, True, False, row_error_found
+        return record, reduce_row, False, False, row_error_found
 
-    def writeToStaging(self, record, jobId, submissionId, passedValidations, interfaces, writer, rowNumber, fileType):
+    def writeToStaging(self, record, job_id, submission_id, passed_validations, interfaces, writer, row_number, file_type, error_list):
         """ Write this record to the staging tables
 
         Args:
             record: Record to be written
-            jobId: ID of current job
-            submissionId: ID of current submission
-            passedValidations: True if record has not failed first validations
+            job_id: ID of current job
+            submission_id: ID of current submission
+            passed_validations: True if record has not failed first validations
             interfaces: InterfaceHolder object
             writer: CsvWriter object
-            rowNumber: Current row number
-            fileType: Type of file for current job
+            row_number: Current row number
+            file_type: Type of file for current job
+            error_list: list of errors found during this validation
 
         Returns:
             Boolean indicating whether to skip current row
         """
         stagingInterface = interfaces.stagingDb
-        errorInterface = interfaces.errorDb
         try:
-            record["job_id"] = jobId
-            record["submission_id"] = submissionId
-            record["valid_record"] = passedValidations
-            stagingInterface.insertSubmissionRecordByFileType(record, fileType)
-        except ResponseException as e:
+            record["job_id"] = job_id
+            record["submission_id"] = submission_id
+            record["valid_record"] = passed_validations
+            stagingInterface.insertSubmissionRecordByFileType(record, file_type)
+        except ResponseException:
             # Write failed, move to next record
-            writer.write(["Formatting Error", ValidationError.writeErrorMsg, str(rowNumber),""])
-            errorInterface.recordRowError(jobId, self.filename,
-                "Formatting Error",ValidationError.writeError, rowNumber,severity_id=interfaces.validationDb.getRuleSeverityId("fatal"))
+            writer.write(["Formatting Error", ValidationError.writeErrorMsg, str(row_number),""])
```

Comment:
Old code again, but I don't think we need to cast `row_number` to str because it looks like the `writer.write` method is doing that.